conf.py

```diff
@@ -18,6 +20,38 @@
 ### Extensions
 ############################################################
 
+#############################################################
+# Display the contributor
+
+def get_contributors_for_file(github_url, github_folder, pagename, page_source_suffix):
+    edit_url = f"{github_url}/edit/{pagename}{page_source_suffix}"
+    parsed_url = urlparse(edit_url)
+    path_parts = parsed_url.path.split('/')
+    username = path_parts[1]
+    repository = path_parts[2]
+    filename = f"{pagename}{page_source_suffix}"
+    
+    url = f"https://api.github.com/repos/{username}/{repository}/commits?path={filename}"
+    response = requests.get(url)
+    try:
+        response.raise_for_status()
+        contributors = response.json()
+        contributors_dict = {}
+        for contributor in contributors:
+            name = contributor['commit']['author']['name']
+            committer_username = contributor['committer']['login']
+            committer_github_page = f"https://github.com/{committer_username}"
+            if name not in contributors_dict:
```

Comment:
I like this! :slightly_smiling_face: 
I was wondering how to make sure we don't have the same names in the list several times.